src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/SpringAPIServiceTest.kt

```diff
@@ -43,6 +45,6 @@ import kotlin.annotation.AnnotationTarget.CLASS
   SentencingAdjustmentsApiExtension::class,
 )
 @ActiveProfiles("test")
-@SpringBootTest(classes = [WebClientConfiguration::class, WebClientAutoConfiguration::class, ReactiveOAuth2ClientAutoConfiguration::class, ReactiveSecurityAutoConfiguration::class])
+@SpringBootTest(classes = [JacksonAutoConfiguration::class, CodecsAutoConfiguration::class, WebClientConfiguration::class, WebClientAutoConfiguration::class, ReactiveOAuth2ClientAutoConfiguration::class, ReactiveSecurityAutoConfiguration::class])
```

Comment:
Some magic to make sure Spring wires up the WebClient.Builder correctly for these tests.